internal/env.go

```diff
@@ -17,9 +18,14 @@ const (
 	DatadogAPMEndpoint = "DD_TRACE_AGENT_URL"
 	// DatadogEnableExtraProfiling is the environment variable key for whether to enable extra profiling or not.
 	DatadogEnableExtraProfiling = "DD_ENABLE_EXTRA_PROFILING"
+	// DatadogEnable is the environment variable key for whether to enable the Datadog integration.
+	DatadogEnable = "DD_ENABLE"
 )
 
-// IsDatadogConfigured checks some common environment-variables to determine if the service is configured to use Datadog.
+// IsDatadogConfigured checks some common environment-variables to determine if
+// the service is configured to use Datadog.
+//
+// Deprecated: Use IsDatadogEnabled()
```

Comment:
Do we really need to deprecate this function?
It would be a smoother transition for users if we simply changed the implementation of this function instead of adding a new function. The body of `IsDatadogEnabled` can be moved to this function instead.

Reply:
https://github.com/coopnorge/go-datadog-lib/pull/323/files#diff-418696ec24e0293d68f7a77cb935635f7e46e1e7c85e6d78520c19f48845eca4R25-R41

Reply:
I realised that this function is in a internal package should not be used outside of the module, deleting it.